pypdf/_text_extraction/_layout_mode/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Structured text extraction extension for pypdf"""
```

Comment:
Is "structured text" the same as "layout mode"? Is one of the two terms more common?

Reply:
Replaced all references to "structured" in new object docstrings with references to "layout".